build/jemalloc.m4

```diff
@@ -58,8 +58,8 @@ if test "$has_jemalloc" != "no"; then
   if test "$jemalloc_base_dir" != "/usr"; then
     TS_ADDTO(CPPFLAGS, [-I${jemalloc_include}])
     TS_ADDTO(LDFLAGS, [-L${jemalloc_ldflags}])
-    TS_ADDTO(LDFLAGS, [-Wl,--add-needed -L${jemalloc_ldflags} -Wl,-rpath,${jemalloc_ldflags} -Wl,--no-as-needed])
-    TS_ADDTO_RPATH(${jemalloc_ldflags})
+  dnl  TS_ADDTO(LDFLAGS, [-Wl,--add-needed -L${jemalloc_ldflags} -Wl,-rpath,${jemalloc_ldflags} -Wl,--no-as-needed])
+  dnl  TS_ADDTO_RPATH(${jemalloc_ldflags})
```

Comment:
Did you mean to include this, was it for testing only?

Reply:
Oh, just testing so I could use a debug jemalloc.  I'll fix it. Thanks!